src/screens/Receive.js

```diff
@@ -27,8 +27,8 @@ class ReceiveScreen extends React.Component {
       // eslint thinks routes is not used, but TabView uses it
       // eslint-disable-next-line react/no-unused-state
       routes: [
-        { key: 'address', title: t`My Address` },
-        { key: 'paymentRequest', title: t`Payment Request` },
+        { key: 'address', title: t`My Address`.toUpperCase() },
+        { key: 'paymentRequest', title: t`Payment Request`.toUpperCase() },
```

Comment:
Under Tab View `v4` the tab titles are no longer uppercase by default, so this change is necessary to keep the UI.